k9mail/src/main/java/com/fsck/k9/mailstore/LocalFolder.java

```diff
@@ -2040,6 +2038,7 @@ private Account getAccount() {
     }
 
     // Note: The contents of the 'message_parts' table depend on these values.
+    @SuppressWarnings("unused")
```

Comment:
I'm not a fan of suppressing this warning. It's a good reminder that the decision to put these values here needs to be revisited.